mw/html.d.ts

```diff
@@ -31,7 +31,7 @@ declare global {
              */
             function element(
                 name: string,
-                attrs?: Record<string, string>,
+                attrs?: Record<string, string | number | boolean>,
```

Comment:
Attribute values are converted to string, so other simple values may be provided.